dependencies/windows/tools.R

```diff
@@ -119,3 +119,27 @@ ensure_dir <- function(...) {
             fatal("Failed to create directory '%s'\n", dir)
    }))
 }
+
+replace_one_line = function(filepath, orig_line, new_line) {
```

Comment:
I think this might be more simply expressed as:

```r
contents <- readLines(filepath)
replaced <- gsub(contents, orig_line, new_line)
if (!identical(contents, replaced))
  writeLines(replaced, filepath)
```

Reply:
Neato, will change it to that.